addons/account/models/account_reconcile_model.py

```diff
@@ -185,6 +165,25 @@ def _onchange_match_total_amount_param(self):
     # RECONCILIATION PROCESS
     ####################################################
 
+    def _make_line_for_widget(self, line):
```

Comment:
what's that for?

Reply:
The client needs more info than the server to manage the data in the reconciliation widget. This is adding the value needed.